src/styles/blocks/cash.scss

```diff
@@ -0,0 +1,20 @@
+.cash {
+  display: flex;
+  height: 19px;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 16px;
+}
+
+.cash__name--price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
```

Comment:
Consider defining this color value as a variable in a separate file to adhere to the task requirements for maintainability.